src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4155,6 +4398,23 @@ namespace AirflowNetworkBalanceManager {
 		static bool OneTimeFlag( true );
 		static bool ErrorsFound( false );
 		Real64 GlobalOpenFactor;
+		Real64 ZonePressure1;
```

Comment:
@mjwitte @Myoldmopar The local variables have no complains from CI. If the local variables are initialized once, I am OK to set up values. However, if I add a statement which is executed every time, it may not need.